test/e2e/suite-ingress/suite-ingress-features/status.go

```diff
@@ -275,6 +275,20 @@ wrw7im4TNSAdwVX4Y1F4svJ2as5SJn5QYGAzXDixNuwzXYrpP9rzA2s=
 		assert.Contains(ginkgo.GinkgoT(), output, `status: "False"`, "status.conditions.status  is recorded")
 	})
 
+	ginkgo.It("check ApisixClusterConfig status is recorded", func() {
+		// create ApisixClusterConfig resource
+		clusterConfigName := "cluster-config-name"
+		assert.Nil(ginkgo.GinkgoT(), s.NewApisixClusterConfig(clusterConfigName, true, true), "create cluster config error")
```

Comment:
Hey @Chever-John did you check the function definition of NewApisixClus...
https://github.com/apache/apisix-ingress-controller/blob/2b31690fc1c4a72c211557f05dbc569bebc60246/pkg/kube/apisix_cluster_config.go#L147

This takes in only obj type.

Reply:
Did u mean I lose e2e-test for this api(beta3)?

Reply:
LGTM!

Reply:
Thanks for pointing out!